packages/miniflare/src/index.ts

```diff
@@ -1155,6 +1155,7 @@ export class Miniflare {
     const forward = new Request(input, init);
     const url = new URL(forward.url);
     forward.headers.set(CoreHeaders.ORIGINAL_URL, url.toString());
+    forward.headers.set(CoreHeaders.DISABLE_PRETTY_ERROR, "");
```

Comment:
This was my dilemma. If we give stringified values, we should infer them as booleans, else "false" would be considered truthy. And then we're faced with a double negative. So, what do you think of my suggestion above (PRETTY_ERROR instead of DISABLE_PRETTY_ERROR with values cast/inferred via `Boolean()`)?

Reply:
🤦